src/migrations/migrations-truffle-5/4_deploy_Pool.js

```diff
@@ -33,18 +34,24 @@ async function migrate({
     const contractArtFilePaths = [
       "@gnosis.pm/dx-contracts/build/contracts/DutchExchangeProxy.json",
       "@gnosis.pm/util-contracts/build/contracts/EtherToken.json",
-      "@gnosis.pm/gno-token/build/contracts/TokenGNO.json",
+      process.env.TOKEN_B_ADDRESS ? "@gnosis.pm/util-contracts/build/contracts/HumanFriendlyToken.json" : "@gnosis.pm/gno-token/build/contracts/TokenGNO.json",
```

Comment:
Is there a way to require this as a migration argument rather than an env variable? Might be more explicit. I'd even argue we should always specify the token (rather than having an implicit default).

Same should probably be true for TRADING_TIME

Reply:
I agree with point 2, not sure about 1 but not really too much on either side of it. I'll leave it as an env for now unless @anxolin also agrees

Reply:
I agree on not using the default. 

About ENV I don't mind that much. U can choose whatever u find more suitable as long as the instructions are included in the README